tests/foreman/ui/test_computeresource_ec2.py

```diff
@@ -65,6 +65,7 @@ def test_positive_default_end_to_end_with_custom_profile(
     :CaseLevel: Integration
 
     :BZ: 1451626
+    :BZ: 2032530
```

Comment:
`:BZ: 1451626, 2032530` is how its done elsewhere, not sure if testimony wouldn't override the first occurrence with the second

Reply:
Good point, thanks!